src/haconiwa/scan/analyzer.py

```
@@ -0,0 +1,302 @@
+                        break
+                
+                # Load config if available
+                if file in self.config_files and file.endswith(('.json', '.yaml', '.yml')):
+                    try:
+                        if file.endswith('.json'):
+                            with open(file_path, 'r') as f:
+                                model_info['config'] = json.load(f)
+                        elif file.endswith(('.yaml', '.yml')):
+                            with open(file_path, 'r') as f:
+                                model_info['config'] = yaml.safe_load(f)
+                    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)